### PR TITLE
Adding `whos-there` from PyPI

### PR DESCRIPTION
**Pull request by sugatoray**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged. `NA`
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sugatoray** [comment]
PR is Ready. Request for review.

cc: @conda-forge/help-python

**Comment by sugatoray** [comment]
## Reason for window build failure

PyTorch is only available for `osx` and `linux` on conda-forge. [(error)](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=432751&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=06d62fd2-ab49-5add-9ee5-83b6882d7d2b&l=1036)

```sh
nothing provides pytorch >=1.6 needed by pytorch-lightning-1.5.0-pyhd8ed1ab_0
```

File: recipes/whos-there/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<4.0
```
Comment (jaimergp) [inline recipes/whos-there/meta.yaml:21]: Are we adding upper bounds now @conda-forge/core ?

Comment (beckermr) [reply recipes/whos-there/meta.yaml:21]: No, we should remove this. I've seen more of this lately. We may want to adjust the linter.

Comment (sugatoray) [reply recipes/whos-there/meta.yaml:21]: If you use grayskull, it does automatically uses `python >=3.x,<4.0` at times.

Comment (sugatoray) [reply recipes/whos-there/meta.yaml]: Removed the upper bound.

**Pull request merged**